docroot/themes/humsci/humsci_basic/backstop/scenarios.json

```diff
@@ -0,0 +1,102 @@
+{
+  "scenarios": [
+    {
+    "label": "Customize site",
+    "url": "http://swshumsci.suhumsci.loc/site-building/customize-site",
+    "referenceUrl": "https://swshumsci-prod.stanford.edu/site-building/customize-site",
+    "delay": 0,
+    "requireSameDimensions": true
+    },
+    {
+      "label": "Tickets",
+      "url": "http://swshumsci.suhumsci.loc/tickets",
+      "referenceUrl": "https://swshumsci-prod.stanford.edu/tickets",
+      "delay": 0,
+      "requireSameDimensions": true
+    },
+    {
+      "label": "collab",
+      "url": "http://swshumsci.suhumsci.loc/collaboration/teams",
+      "referenceUrl": "https://swshumsci-prod.stanford.edu/collaboration/teams",
+      "delay": 0,
+      "requireSameDimensions": true
+    },
+    {
+      "label": "ical feed",
+      "url": "http://swshumsci.suhumsci.loc/site-building/setting-ical-feed",
+      "referenceUrl": "https://swshumsci-prod.stanford.edu/site-building/setting-ical-feed",
+      "delay": 0,
+      "requireSameDimensions": true
+    },
+    {
+      "label": "home",
+      "url": "http://swshumsci.suhumsci.loc/",
+      "referenceUrl": "https://swshumsci-prod.stanford.edu/",
+      "delay": 0,
+      "requireSameDimensions": true
+    },
+    {
+      "label": "impact",
+      "url": "http://swshumsci.suhumsci.loc/legacy",
+      "referenceUrl": "https://swshumsci-prod.stanford.edu/legacy",
+      "delay": 0,
+      "requireSameDimensions": true
+    },
+    {
+      "label": "QA Content",
```

Comment:
The scenarios for Econ did not appear in the results of my BackstopJS Report. I only saw the files for the default site. According to the `readme` instructions, new scenarios are added to the `backstop/scenarios.json` file. 
 I'm curious as to what this json file used for?

Reply:
Yeah.. I was hoping we'd be able to import a separate json file into here, but it wasn't possible, and now a separate file is just confusing. I'll delete scenarios.json and add all scenarios here.